### PR TITLE
Bind address to 0,0,0,0 to allow prometheus scraping

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: CAPI/KubeadmControlplane.yaml [hunk 1]
```diff
@@ -24,3 +24,27 @@ spec:
...
+    match:
+      resources:
+        kind: 
+        - controlplane.cluster.x-k8s.io/v1alpha3/KubeadmControlPlane
+    context:
+    - name: clusters
+      apiCall:
+        urlPath: "/apis/cluster.x-k8s.io/v1alpha3/namespaces/{{ request.object.metadata.namespace }}/clusters"
+        jmesPath: "items[*]"
+    mutate:
```
Comment (MarcelMue) [inline CAPI/KubeadmControlplane.yaml:37]: I think you can just put this in the same mutate as above (I think)

Comment (QuentinBisson) [reply CAPI/KubeadmControlplane.yaml:37]: Probably, but naming :D

Comment (QuentinBisson) [reply CAPI/KubeadmControlplane.yaml:37]: I prefer it to not become too convoluted as well

File: CAPI/KubeadmConfig.yaml [hunk 2]
```diff
@@ -24,3 +24,21 @@ spec:
           labels:
             +(cluster.x-k8s.io/watch-filter): "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-bootstrap-provider-kubeadm'].version | [0]}}"
             cluster.x-k8s.io/watch-filter: "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-bootstrap-provider-kubeadm'].version | [0]}}"
+  - name: kubeadmconfig-metrics
+    match:
+      resources:
+        kinds:
+        - bootstrap.cluster.x-k8s.io/v1alpha3/KubeadmConfig
+    context:
```
Comment (MarcelMue) [inline CAPI/KubeadmConfig.yaml:32]: Don't think you need any context, as it's not used

File: CAPI/KubeadmControlplane.yaml (hunk 1 above)
Comment (QuentinBisson) [reply CAPI/KubeadmControlplane.yaml]: I tried to simplify it, let's see if it works first :D

File: CAPI/KubeadmConfig.yaml (hunk 2 above)
Comment (QuentinBisson) [reply CAPI/KubeadmConfig.yaml]: Most likely no :D

**Pull request merged**